scripts/scripts.js

```diff
@@ -54,7 +54,11 @@ export function domEl(tag, ...items) {
     items = rest;
 
     Object.entries(attributes).forEach(([key, value]) => {
-      if (!key.startsWith('on')) {
+      // Add additional properties here for things that fail to reflect through setAttributes
+      const directAttributes = ['innerHTML', 'innerText'];
```

Comment:
@synox I found that when attempting to include an `innerHTML` property through the `domEl` wrapper functions (such as `a()` or `span()` that the attribute would be passed through in lower case and thus not render in the page.  

I played around with different approaches to try and combat that but even when the case was correct I found that the provided text did not render.  So I instead opted to have a specific list of attributes that instead of being set via `setAttribute` are instead applied directly through bracket notation to the element being created.  

If that's a horrible idea for any number of reasons let me know and I can revert and go back to my previous approach of jkust manually setting the desired innerHTML after the element has been created.

Reply:
That's a good idea. Looks fine how you implemented it. I don't see why you would need innerText though, as you can add text as the last argument.

Reply:
I didn't know that (and would have circumvented the need for this approach).  But yeah I'll pull out the innerText if it's unnecessary (I was using innerHTML anyway).